### PR TITLE
Stock parts now build 5x faster

### PR DESCRIPTION
**Pull request by ghost**
All manipulators, capacitors, matter bins, scanning modules, micro-lasers now build 5 times faster in the Protolathe.

Credit to https://github.com/ParadiseSS13/Paradise/pull/4633 for the code, I just changed 5 to 0.2 to work on TG

:cl:
tweak: Protolathe stock parts now build 5 times faster
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
:+1: only science buff I can get behind

**Comment by ShadowDeath6** [comment]
good

**Comment by Fox-McCloud** [comment]
I'd do this for cells, too (which are also technically stock parts).

**Comment by uraniummeltdown** [comment]
>I'd do this for cells, too (which are also technically stock parts).
While cells are used in a few machines like the Porta Chem Dispenser and SMES, they're not really as important as the other stock parts, Scientists don't really need to mass print cells as much as the other stock parts.

**Comment by Aloraydrel** [comment]
This kill tactical waiting

**Comment by GunHog** [comment]
Very good change!

**Comment by XDTM** [comment]
And here i was thinking people'd shoot it down due to it being an RnD buff

**Comment by LaharlMontogmmery** [comment]
Anyone who did RnD knows how tedious it is waiting for parts to print.

I approve.

File: code/modules/research/designs/stock_parts_designs.dm [hunk 1]
```diff
@@ -32,6 +32,7 @@
 	materials = list(MAT_METAL = 100, MAT_GLASS = 100)
 	build_path = /obj/item/weapon/stock_parts/capacitor
 	category = list("Stock Parts","Machinery","initial")
+	lathe_time_factor = 0.2
```
Comment (GunHog) [inline code/modules/research/designs/stock_parts_designs.dm]: Recommendation: Make `/datum/design/stock_parts/` as a parent for all of the stock parts, so you only need to set `lathe_time_factor = 0.2` once, and should you need to change it, you only need it in one place. (copy pasta rule)

**Pull request merged**